framework/src/controller/action.ts

```diff
@@ -32,56 +33,72 @@ export interface ActionsObject {
 	[key: string]: Action;
 }
 
+type ID = string | number | null;
+
 export class Action {
-	public module: string;
-	public name: string;
-	public handler?: (action: ActionInfoObject) => unknown;
-	public source?: string;
+	public jsonrpc = VERSION;
+	public id: ID;
+	public method: string;
 	public params: object;
+	public source?: string;
+	public handler?: (action: ActionInfoObject) => unknown;
 
 	public constructor(
-		name: string,
+		id: ID,
+		method: string,
```

Comment:
I don't think changing `name` to `method` or removing `module` attribute is appropriate here. In framework we still call and should call it `action name` and `action.module`. The change should be translated in `toJsonRPC` function where ``method: `${module}:${name}` ``